rke/k8s_rke_system_images.go

```diff
@@ -8522,5 +8522,137 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			IngressBackend:            m("k8s.gcr.io/defaultbackend:1.4"),
 			MetricsServer:             m("gcr.io/google_containers/metrics-server-amd64:v0.3.1"),
 		},
+		// Enabled out of band post v2.7.0
+		"v1.22.16-rancher1-1": {
```

Comment:
We group by minor release, so `v1.22.16-rancher1-1` definition should follow the previous `v1.22.x-ranchery-z` definition. The same for newly introduced 1.23 and 1.24.

E.g. like here: https://github.com/rancher/kontainer-driver-metadata/pull/967

Reply:
done